articles/watch-your-state.md

```diff
@@ -0,0 +1,49 @@
+---
+authorName: Raphael M. Reischuk
+authorGithubUsername: reischuk
+issue: 54
+title: Watch Your State
+---
+# {{page.title}}
+
+
+In May 1960, the legendary Pininfarina-designed Peugeot 404 was presented to the public. Its manufacturing process was controlled by manually curated lists in which each car in the production pipeline had its particular entry. Car by car was assembled according to a fixed plan. Consequently, spontaneous adaptations or dynamic modifications of the production plan were impossible when components were suddenly missing, or when experts responsible for certain tasks got sick.
```

Comment:
In May 1960, the legendary Pininfarina-designed Peugeot 404 was presented to the public. -> In May 1960, the legendary Peugeot 404 was presented to the public. // der Designer scheint mir nicht so wichtig

Reply:
fixed